### ISSUE TITLE
Is revel dead?

### ISSUE THREAD

**Issue by mandrean** [revel/revel#1119]
Is this project dead? No activity for like 8 months...

Theres over 30 pull requests waiting for feedback or to be merged.

So sad.

**Comment by notzippy** [comment]
If you look the PR's have been tagged and labeled, the product is production ready as it is. I would love to be able to allocate time to this project but I don't have the bandwidth. @brendensoares is leading the development on this with as much time as he can allocate. If you have time to help contact him.

**Comment by torden** [comment]
Not long ago, I sent email to @jeevatkm, however his reply yet until now.
As I knew, This project maintainer are @brendensoares, @pedromorgan @jeevatkm, 
31 PRs waiting for them.

**Comment by runner-mei** [comment]
or add new member

**Comment by kgodwin** [comment]
Yeah, if this goes through the end of Feb without activity we probably need to fork this after this much inactivity. No offense to anyone but things like this need a certain minimum of activity to avoid death.

**Comment by shawncatz** [comment]
@brendensoares @pedromorgan @jeevatkm I'm willing to help... I'd rather see more people added than fork and diverge

**Comment by pedromorgan** [comment]
I'm just the docs maintainer, Its mainly up to @brendensoares as he's the BDFL

**Comment by lujiacn** [comment]
What is next action? Just waiting for another couple of weeks?

**Comment by shawncatz** [comment]
a fair question. 

there hasn't been a code commit/merge to master in 9 months. 
@brendensoares doesn't seem to be super active on github. 
@jeevatkm appears to have started working on a different web framework go-aah/aah. 
and neither of them have responded to this issue

**Comment by pedromorgan** [comment]
I'm trying to get in contact also!

**Comment by pedromorgan** [comment]
Ok here's a suggestion (note my contrib is the docs as an end user, not a golang expert)

- I'm trying to get in touch with @jeevatkm and @brendensoares. 
- In the meantime will need a list of Contributors to add to project. 
- If I dont hear back soon, then I'll start adding people to the project. 
- BUT, I dont wanna upset anyone!!

**Comment by pedromorgan** [comment]
Forget above, @brendensoares is the owner, and the only one who can add users :-((((((((((((((((((

**Comment by notzippy** [comment]
I talked with Brenden on IRC and he is still around, try reaching him there.

**Comment by jeevatkm** [comment]
My apologize for the delayed response.  As @shawncatz mentioned, Yes I have started working on aah web framework for Go. I felt I could do more to the web framework and Go community.

I don't have permission to add contributors, we have to touch base with @brendensoares. Only he can help.

**Comment by pedromorgan** [comment]
Maybe we start adding "reviews" to pull requests, and if it gets more thatn say 2, then its merged ??

**Comment by notzippy** [comment]
I am thinking most features can be merged, as long as it is not to much like spaghetti code. I really liked the PR to allow different template engines..

**Comment by brendensoares** [comment]
I'm on IRC every day and I answer questions there. I have no time to do actual coding that's why I try to organize core members to assist with actual contributions. Lately I've not even made time to stay on top of github issues.

Forking this repo would be a logistical mistake. Adding more community members to the core team to give them the ability to make decisions as a core team is the correct way forward. The biggest issue with open source project management is getting a consistent core team to stick around. It's like herding cats mostly because people have families, friends, and jobs that are more important.

It would be great if we could get corporate sponsors as well. I have recently formed a corporation in the US that I do my consulting under and am planning on sponsoring development through that, but there are still lots of variables. For example, if paying core team members for their work good or would it actually have no significant effect? Would offering enterprise support allow us to fund a full time developer or two? Is there demand there?

Just some background, @jeevatkm and others have done wonders for Revel recently. Many past core members have gone off to their own web framework projects as experiments. Partly because Revel is missing some active development direction.

Other logistical concerns are how often we need to release versus what is reasonably achievable. Also, though there are many PRs, not all of them are consistent with what Revel is and needs to be, so there is serious considerations for effects of changes in the code to the community. This requires discussion, but talking to myself has no benefit when I'm not even actively coding for Revel (again I'm just managing the project at this point).

TL;DR: if you want to join @revel/core step right up! It's not like I haven't tried to organize a team in the past:
- [Open call for core team members](https://groups.google.com/forum/?pli=1#!searchin/revel-framework/open$20call|sort:relevance/revel-framework/6HtUNaXNKEI/oYHI9jN2DgAJ)
- [Open Call for Revel Core Developers](https://groups.google.com/forum/?pli=1#!searchin/revel-framework/open$20call|sort:relevance/revel-framework/ABjt3CzJWoE/j_aJmqH8S1QJ)
- https://github.com/revel/revel/issues/892#issuecomment-84640422
- https://github.com/revel/revel/issues/917
- https://github.com/revel/revel/issues/1051
- https://github.com/revel/revel/issues/816

**Comment by jeevatkm** [comment]
@brendensoares Thank you. I really enjoyed working in Revel, @pedromorgan and you. I'm available on [gitter.im](https://gitter.im)(you can find me using github name) let's be in touch.

**Comment by shawncatz** [comment]
@brendensoares @jeevatkm @pedromorgan I appreciate the responses... my hope was that we could get more involved, so that others didn't feel the need to fork

As I said before, I'm willing to contribute. I'm relatively new to golang (about a year), but have 25+ years of programming experience.

**Comment by notzippy** [comment]
@shawncatz @jeevatkm @pedromorgan  and others, we have scheduled a meeting for tomorrow at 8pm PST, the room is https://appear.in/revelframework . Please join us.

**Comment by shawncatz** [comment]
I’ll be there

-- 
Shawn Catanzarite
408-605-7744
http://shawncatz.com/
http://linkedin.com/in/shawncatanzarite

On March 1, 2017 at 5:22:47 PM, notzippy (notifications@github.com) wrote:

@shawncatz <https://github.com/shawncatz> @jeevatkm
<https://github.com/jeevatkm> @pedromorgan <https://github.com/pedromorgan>
and others, we have scheduled a meeting for tomorrow at 8pm PST, the room
is https://appear.in/revelframework . Please join us.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/revel/revel/issues/1119#issuecomment-283526270>, or mute
the thread

**Comment by kvfi** [comment]
Hey @notzippy, could you please specify the date. Is it March 2nd or 3rd?

**Comment by jeevatkm** [comment]
@notzippy I believe it's March 2nd evening. I will try to join or I will go through the MOM later on. I have to pickup my daughter around that time.

**Comment by brendensoares** [comment]
https://gitter.im/revel?utm_source=share-link&utm_medium=link&utm_campaign=share-link

We're trying slack and gitter for chat along side our IRC channel. Any preferences?

**Comment by brendensoares** [comment]
I think we'll go with gitter as it allows public channels without manually invites. Plus, it integrates well with GH and still has a [mobile app](https://gitter.im/apps).

**Comment by shawncatz** [comment]
yeah agreed, think it's easier

**Comment by brendensoares** [comment]
we'll continue this convo on gitter

**Comment by denysvitali** [comment]
F

**Comment by challarao** [comment]
@brendensoares It was long since this issue was created. May we know the latest status on this?

**Comment by Steve-hyas** [comment]
Still alive, @challarao the go.mod changes are proving to be a little more challenging then anticipated because of the way they were implemented.

**Comment by notzippy** [comment]
Still alive, @challarao the go.mod changes are proving to be a little more challenging then anticipated because of the way they were implemented.
